<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FAccMonthlyReport</class>
 <widget class="QWidget" name="FAccMonthlyReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="EDateEdit" name="deStart">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="EDateEdit" name="deEnd">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Group by</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbDay">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>Day</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbWeek">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>Week</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbMonth">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>Month</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rbYear">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>Year</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Set date for</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="EQComboBox" name="cbYear">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="currentText">
        <string>2017</string>
       </property>
       <item>
        <property name="text">
         <string>2017</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2018</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="EQComboBox" name="cbMonth">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <item>
        <property name="text">
         <string>January</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>February</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mart</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>April</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>May</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>June</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>July</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>August</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>September</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>October</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>November</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>December</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chCanceled">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="text">
        <string>Canceled</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
  <customwidget>
   <class>EQComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">eqcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
